src/envoy/auth/jwt.cc

```diff
@@ -95,20 +93,14 @@ std::string decode(std::string input) {
   return Base64::decode(input);
 }
 
-}  // namespace Base64url
-
-namespace Util {
-
 const uint8_t *castToUChar(const std::string &str) {
   return reinterpret_cast<const uint8_t *>(str.c_str());
 }
 
-}  // namespace Util
-
 bssl::UniquePtr<EVP_PKEY> evpPkeyFromStr(const std::string &pkey_pem) {
```

Comment:
should we following Google code style to have function name started with Upper case if it is camel case?